tools/python_dispatcher.py

```diff
@@ -0,0 +1,161 @@
+import re
+import torch._C as C
```

Comment:
This is nifty. Since you're depending on torch library, I'd suggest putting this Python file inside the `torch/` folder (you can underscore it to make it private--the point is to preserve the invariant that tools Python files work without having a built PyTorch) and adding some simple tests for it so it doesn't bitrot.